### ISSUE TITLE
possibility to combine multiple conditions and multiple actions on one rule

### ISSUE THREAD

**Issue by adminschb01** [wekan/wekan#2953]
Thanks again for the great work.
one of the best feature of wekan is the board rules engine, as it permits the automation of some actions.
it would be better to permit combining multiple contitions (with logic operators and,or)  and actions on one rule.
an other feature is to offer the possibility to use some variables on the contenant of the email action making it more dynamic (like card  title, list name... )

**Comment by FoksVHox** [comment]
I agree. I'll hope that in the next update to the rules, it will be something like this. Another thing that would be beneficial is the option the use a variable for the current swimlane for the card.

**Comment by renhardy** [comment]
I agree this would be a major improvement.

**Comment by Meeques** [comment]
I agree, too.
